package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nextd3",
```

Comment:
We can add `freefrom-map-frontend` here instead

Reply:
done